spark/v3.5/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/parser/extensions/IcebergSparkSqlExtensionsParser.scala

```diff
@@ -123,7 +123,9 @@ class IcebergSparkSqlExtensionsParser(delegate: ParserInterface) extends ParserI
     if (isIcebergCommand(sqlTextAfterSubstitution)) {
       parse(sqlTextAfterSubstitution) { parser => astBuilder.visit(parser.singleStatement()) }.asInstanceOf[LogicalPlan]
     } else {
-      RewriteViewCommands(SparkSession.active).apply(delegate.parsePlan(sqlText))
+      // Iceberg views are not supported in LinkedIn, do not rewrite views
```

Comment:
Spark tests are failing  for spark-3.5 module https://github.com/linkedin/iceberg/actions/runs/20234355878/job/58259521405?pr=224. Can we please fix them.